python/equistore-operations/equistore/operations/add.py

```diff
@@ -33,19 +37,19 @@ def add(A: TensorMap, B: Union[float, TensorMap]) -> TensorMap:
 
     blocks = []
     if isinstance(B, TensorMap):
-        _check_same_keys(A, B, "add")
+        _check_same_keys_raise(A, B, "add")
         for key, block_A in A.items():
             block_B = B[key]
-            _check_blocks(
+            _check_blocks_raise(
                 block_A,
                 block_B,
-                props=["samples", "components", "properties"],
+                check=["samples", "components", "properties"],
```

Comment:
Here you pass a list. Shall we change the typing on all the functions in `_utils.py` to be instead `Sequence[str]` instead of `Tuple[str]`?